[CloudPC] Added MEM admin poweron and poweroff actions (#20365)
microsoftgraph/microsoft-graph-docs: api-reference/beta/api/cloudpc-poweron.md

---
title: "cloudPC: power on"
description: "Power on specific Cloud PC."
author: "rbayetov"
ms.localizationpriority: medium
ms.prod: "cloud-pc"
doc_type: apiPageType
---

# cloudPC: powerOn

Namespace: microsoft.graph

[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]

> Perform power on for a Cloud PC. This is to cover [Frontline](https://www.microsoft.com/en/windows-365/frontline) CloudPC for MEM admin scenarios.

Reviewer: ```suggestion
Perform power on for a Cloud PC. This is to cover [Frontline](https://www.microsoft.com/en/windows-365/frontline) CloudPC for MEM admin scenarios.
```
```suggestion
Power on a Frontline Cloud PC.  This action supports [Frontline](https://www.microsoft.com/en/windows-365/frontline) MEM admin scenarios. 
```

> Power on a Frontline Cloud PC. This action supports [Windows 365 Frontline](https://www.microsoft.com/en/windows-365/frontline) MEM admin scenarios. 

Reviewer 2: Please see my comments on the powerOff topic and apply here as well.

> Power on a Frontline Cloud PC. This action supports [Windows 365 Frontline](https://www.microsoft.com/en/windows-365/frontline) Microsoft Endpoint Manager (MEM) admin scenarios. 

Reviewer: ```suggestion
Power on a [Windows 365 Frontline](https://www.microsoft.com/en/windows-365/frontline) Cloud PC. This action supports Microsoft Endpoint Manager (MEM) admin scenarios. 
```

> CloudPC will be allocated for assigned end user immediately and license will be assigned for Frontline CloudPC. The API only targets IT admin consumers.

Reviewer: ```suggestion
After powering on a Frontline CloudPC, it will be allocated to a user, and licenses will be assigned immediately. Only IT admin users can perform this action. 
```

> After a Frontline Cloud PC is powered on, it is allocated to a user, and licenses are assigned immediately. Only IT admin users can perform this action. 

Reviewer: ```suggestion
After a [Windows 365 Frontline](https://www.microsoft.com/en/windows-365/frontline) Cloud PC is powered on, it is allocated to a user, and licenses are assigned immediately. Only IT admin users can perform this action. 
```
